docker/cli
# vendor.conf: golang.org/x/crypto v0.2.0 (Fix `ssh: parse error in message type 27` with OpenSSH >= 8.9)

Signed-off-by: Akihiro Suda <akihiro.suda.cz@hco.ntt.co.jp>



**- What I did**
Fix `ssh: parse error in message type 27` with OpenSSH >= 8.9

- https://github.com/moby/buildkit/pull/3274
- https://github.com/docker/buildx/pull/1407

**- How I did it**

Bumped up golang.org/x/crypto to include https://github.com/golang/crypto/commit/bd7e27e6170dc8db5a8588e1ac1e971eae90710f

**- How to verify it**

Run `docker build --ssh default` with `DOCKER_BUILDKIT=1`, and make sure no `ssh: parse error in message type 27` error occurs

**- Description for the changelog**

Fix `ssh: parse error in message type 27` with OpenSSH >= 8.9

**- A picture of a cute animal (not mandatory but encouraged)**
🐧
